org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/PersistenceConfiguration.xtend

```diff
@@ -12,6 +12,9 @@ class PersistenceConfiguration extends DropwizardApplicationConfiguration {
 	@Accessors
 	private String remoteRepoUrl
 	
+	@Accessors
+	private String repoConnectionMode = 'pullPush' // 'pullOnly' 
```

Comment:
does it make sense to introduce an enum here? (does dropwizard support that here…?)

Reply:
I'd rather postpone this until we had the discussion/concept about future repository collaboration. I'll check whether this changed can be easily applied.